### PR TITLE
Update docstrings in guide.py and label.py

### PR DESCRIPTION
**Pull request by OLarionova-HORIS**

### CODE REVIEW

File: python-package/lets_plot/plot/guide.py [hunk 1]
```diff
@@ -100,20 +100,24 @@ def guide_colorbar(barwidth=None, barheight=None, nbin=None):
 def _guide(name, **kwargs):
     return FeatureSpec('guide', name=name, **kwargs)
 
-def guides(**kwargs):
+
+def guides(**scale_guides):
```
Comment (alshan) [inline python-package/lets_plot/plot/guide.py:104]: Lets use name "kwargs"

File: python-package/lets_plot/plot/guide.py [hunk 2]
```diff
@@ -100,20 +100,24 @@ def guide_colorbar(barwidth=None, barheight=None, nbin=None):
...
     """
     Set guides for each scale.
 
     Parameters
     ----------
-    kwargs
-        Name-guide pairs where name should be an aesthetic.
-        The guide should be a call to a guide function, i.e. `guide_colorbar()` or `guide_legend()`.
+    scale_guides
+        Arguments to specify scale guides.
```
Comment (alshan) [inline python-package/lets_plot/plot/guide.py:111]: Prev description (i.e. "Name-guide pairs where name should be an aesthetic.") sounds better. 

It's not quite clear, what is "scale name".

File: python-package/lets_plot/plot/guide.py [hunk 3]
```diff
@@ -137,4 +141,4 @@ def guides(**kwargs):
                    color=guide_colorbar(nbin=8, barwidth=20))
 
     """
-    return FeatureSpec('guides', name=None, **kwargs)
\ No newline at end of file
+    return FeatureSpec('guides', name=None, **scale_guides)
```
Comment (alshan) [inline python-package/lets_plot/plot/guide.py:144]: Lets use name "kwargs".

File: python-package/lets_plot/plot/label.py [hunk 4]
```diff
@@ -110,20 +110,27 @@ def ylab(label):
...
-        e.g. `title='Plot title'` or `aesthetic='Scale label'`.
+    title : str
+        The text for the plot title.
+    subtitle : str
+        The text for the plot subtitle.
+    caption : str
+        The text for the plot caption.
+    labels
+        Arguments to specify labels.
+        The argument name corresponds to the aesthetic name.
```
Comment (alshan) [inline python-package/lets_plot/plot/label.py:127]: Prev variant (i.e. "Name-value pairs where name should be an aesthetic and..") sounds better.
"The argument name.." is weird.

File: python-package/lets_plot/plot/guide.py (hunk 3 above)
Comment (OLarionova-HORIS) [reply python-package/lets_plot/plot/guide.py:144]: Fixed: https://github.com/JetBrains/lets-plot/commit/b17bb25961012231b36915b0e50add13129e9f9e

**Pull request merged**